azure-devops/azext_devops/dev/common/format.py

```diff
@@ -0,0 +1,34 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def trim_for_display(text, max_length):
+    if text is None:
+        return 'None'
```

Comment:
This is a generic util function. It should not turn empty string to 'None' which might have a different meaning in some context.

Reply:
agree, returning the input as is when it is falsy